src/controller/AdminController.php

```diff
@@ -128,7 +129,7 @@ public function displayInvalidComments(UserModel $userModel, PostModel $postMode
 		$invalideComments = $commentModel->loadInvalidComments();
 
 		// display 
-		$this->render('admin_waiting_comments.twig', $_SESSION, [], [], $invalideComments);
+		$this->render('admin_waiting_comments.twig', [], [], [], $invalideComments, [], $_SESSION);
```

Comment:
```suggestion
		$this->render('admin_waiting_comments.twig', $_SESSION, [], [], $invalideComments);
```

Reply:
Modif Ok : 
$this->render('admin_waiting_comments.twig', $_SESSION, [], [], [], $invalideComments, []);